test/examples_test.go

```diff
@@ -135,7 +142,7 @@ func exampleTest(path string, waitValidateFunc waitFunc, kind string) func(t *te
 			t.Skipf("Couldn't substitute environment: %v", err)
 		}
 
-		out, err := koCreate(subbedInput, namespace)
```

Comment:
Super tiny nit: there's a comment on line 151, `// Parse from koCreate for now` that I think is no longer right with this change?  Maybe `// Parse from createFunc for now`?